dorm-room/1.1.2022.md

```diff
@@ -22,8 +22,8 @@
 - @Manoj Sarna earned his diploma in mechanical engg after listening to all experiences from mechanical engg.
 - @Aman Sandeep Katyal wants to learn about R&D of semiconductors and how chips are manufactured. New startup discussion.
 - Now, International level discussion regarding semiconductor manufacturing between @Aman Sandeep Katyal , @Rishabh Rathore , @Abhi Jain  and many more
-- Discussing about KYJUS dev jobs and poking HatJr hahah. This turned into deep discussion about all the unicorn hypothetical stuff thats carried out in Kyjus and WHJ. And also some work experience.
-People are discussing so much about KYJUS that mostly we will end up investing in KYJUS stocks or something. END OF TOPIC
+- Discussing about KYJUS dev jobs and poking SafedTopee hahah. This turned into deep discussion about all the unicorn hypothetical stuff thats carried out in Kyjus and WHJ. And also some work experience.
+People are discussing so much about SafedTopee that mostly we will end up investing in KYJUS stocks or something. END OF TOPIC
```

Comment:
In line 26 it should be KYJUS instead of SafedTopee

Reply:
My bad.

Reply:
Just make another commit to this same branch, it will reflect in the PR